### PR TITLE
feat(plugin-calendar): Update whistler endpoint

### PR DESCRIPTION
**Pull request by urbenlegend**
Update plugin-calendar to use new whistler api

Can test here: https://sqbu-github.cisco.com/pages/benxiao/web-client/web/

@saurjai3

### CODE REVIEW

File: packages/node_modules/@ciscospark/internal-plugin-calendar/src/index.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ registerInternalPlugin(`calendar`, Calendar, {
         name: `transformMeeting`,
         direction: `inbound`,
         fn(ctx, object) {
+          // console.info(`TRANSFORM OBJECT\n${JSON.stringify(object, null, 4)}`);
```
Comment (saurjai3) [inline packages/node_modules/@ciscospark/internal-plugin-calendar/src/index.js]: commented code can be removed.

File: packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js [hunk 2]
```diff
@@ -20,81 +19,70 @@ import {merge} from 'lodash';
...
+        responseType: `UNKNOWN_RESPONSE`,
+        participantType: `REQUIRED`,
+        orgId: params.participants.participant2.orgId,
+        encryptedEmailAddress: params.participants.participant2.emailAddress,
+        encryptedName: params.participants.participant2.name
+      }
+    ]
+  };
+
+  console.info(`PAYLOAD: ${JSON.stringify(payload, null, 4)}`);
```
Comment (saurjai3) [inline packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js]: console statement can be removed.

File: packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js [hunk 3]
```diff
@@ -20,81 +19,70 @@ import {merge} from 'lodash';
...
     body: payload,
     qs: {
-      isinteg: typeof process.env.WDM_SERVICE_URL !== `undefined` && process.env.WDM_SERVICE_URL.indexOf(`wdm-a.wbx2.com`) < 0
+      changeType: type,
+      useProduction: typeof process.env.WDM_SERVICE_URL === `undefined` || process.env.WDM_SERVICE_URL.indexOf(`wdm-a.wbx2.com`) >= 0
     }
   })
     .then((res) => {
       // Test response is using our parameters
+      console.info(`WHISTLER RESPONSE\n${JSON.stringify(res, null, 4)}`);
```
Comment (saurjai3) [inline packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js]: console statement can be removed.

File: packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js [hunk 4]
```diff
@@ -121,6 +109,7 @@ describe(`plugin-calendar`, () => {
         .then((users) => {
           [creator, spock, mccoy] = users;
 
+          console.info(`CREATOR\n${JSON.stringify(creator, null, 4)}`);
```
Comment (saurjai3) [inline packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js]: console statement can be removed.

File: packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js [hunk 5]
```diff
@@ -169,13 +158,16 @@ describe(`plugin-calendar`, () => {
...
         };
 
-        return postToWhistler(spark, makeMockMeetingPayload(meetingParams))
+        return postToWhistler(spark, `CREATE`, makeMockMeetingPayload(meetingParams))
           .then((res) => {
             const createdMeeting = res.body;
             return spark.internal.calendar.list()
               .then((meetings) => {
-                const testMeeting = meetings.find((meeting) => meeting.seriesId === createdMeeting.meeting.icalUid);
+                console.info(`MEETING LIST\n${JSON.stringify(meetings, null, 4)}`);
```
Comment (saurjai3) [inline packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js]: console statement can be removed.

File: packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js [hunk 6]
```diff
@@ -169,13 +158,16 @@ describe(`plugin-calendar`, () => {
...
+                const testMeeting = meetings.find((meeting) => meeting.seriesId === createdMeeting.meeting.meetingSeriesId);
+                console.info(`SELECTED MEETING\n${JSON.stringify(testMeeting, null, 4)}`);
```
Comment (saurjai3) [inline packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js]: console statement can be removed.

File: packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js [hunk 7]
```diff
@@ -223,12 +224,13 @@ describe(`plugin-calendar`, () => {
...
         });
 
-        return postToWhistler(spark, makeMockMeetingPayload(meetingParams))
+        return postToWhistler(spark, `CREATE`, makeMockMeetingPayload(meetingParams))
           .then((res) => {
             const createdMeeting = res.body;
             return assert.isFulfilled(mercuryPromise)
               .then((calendarMeetingExternal) => {
-                assert.equal(createdMeeting.meeting.icalUid, calendarMeetingExternal.seriesId);
+                console.info(`MERCURY MESSAGE\n${JSON.stringify(calendarMeetingExternal, null, 4)}`);
```
Comment (saurjai3) [inline packages/node_modules/@ciscospark/internal-plugin-calendar/test/integration/spec/calendar.js]: console statement can be removed.

**Comment by urbenlegend** [comment]
Comments addressed

**Pull request merged**